version.py

```diff
@@ -40,18 +40,27 @@
 
 
 def call_git_describe(prefix='', cwd='.'):
-    cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    version_name_cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    last_tag_cmd = 'git describe --tags --abbrev=0'
     try:
         version = subprocess.run(
-            cmd.split(),
+            version_name_cmd.split(),
             stdout=subprocess.PIPE,
             check=True,
         ).stdout.decode()
+
+        last_tag = subprocess.run(
+            last_tag_cmd.split(),
+            stdout=subprocess.PIPE,
+            check=True,
```

Comment:
We're not relying on any shell features. It would be better to run both commands with `shell=False`.

Reply:
Done

Reply:
Did you push all your changes? I don't see `shell=False` in the code.

Reply:
@cameel 
Yes, I have changed it and I pushed everything. See: https://github.com/golemfactory/golem-messages/pull/201/files#diff-78c8de9f11870b74f87e1deeffeccb84R49

Reply:
I have read that you want me to change `check`. Already fixed. Done